ui/src/main/webapp/src/app/project/migration-project.service.ts

```diff
@@ -42,7 +46,8 @@ export class MigrationProjectService extends AbstractService {
 
         if (event.isTypeOf(NewExecutionStartedEvent)) {
             monitoredProject.executions.push((<NewExecutionStartedEvent>event).execution);
-        } else if (event.isTypeOf(ExecutionUpdatedEvent)) {
+        }
+        else if (event.isTypeOf(ExecutionUpdatedEvent)) {
```

Comment:
It resembles me how we can't have an fav icon because it's a convention. my2c, YMMV.